frontend/src/js/controllers/authCtrl.js

```diff
@@ -36,6 +36,10 @@
         vm.isValid = {};
         vm.confirmMsg = '';
         $rootScope.loaderTitle = '';
+        $rootScope.passwordInputType = 'password';
+        $rootScope.confirmPasswordInputType = 'password';
+        $rootScope.showEyeIcon = false;
```

Comment:
You might not need showEyeIcon anymore as you can toggle using the canShowPassword variable.

Reply:
done.